.travis.yml

```diff
@@ -42,7 +42,7 @@ services:
 - xvfb
 install:
 - pip install -r requirements/dev.txt
-- pip install awscli==1.16.57 coveralls
+- pip install --upgrade awscli boto3 botocore moto
```

Comment:
update boto3, botocore, moto in requirements/dev.txt as well

Reply:
Is it okay if the changes are made in requirements/common.txt? 
https://github.com/Cloud-CV/EvalAI/blob/master/requirements/common.txt#L2-L3

Reply:
yes. We need changes in both